### DISCUSSION TITLE
What is the difference between using `vite build` and `remix vite:build`?

### DISCUSSION THREAD

**Discussion by moltar** [remix-run/remix#8895] [Q&A]
These tasks were added in https://github.com/remix-run/remix/pull/8211

And they wrap Vite build programmatically, but do not seem to add any extra functionality from the quick glance.

Is it recommended to use `remix vite:build` commands or can we use `vite build` directly?

**Comment by alessandrojcm** [comment]
I am actually wondering the same thing as I want to control my build programatically. Reading the [source code](https://github.com/remix-run/remix/blob/main/packages/remix-dev/vite/build.ts#L292) it seems that it runs a parallel build for the server bundles so as long as you don't use multiple server bundles it should be fine to run `vite build` I think? But I would like some clarification as well. Ideally I'd like for the build function to be exported from `@remix-run/dev` .
